firmware/controllers/global_shared.h

```diff
@@ -14,11 +14,6 @@
  * The following obscurantism is a hack to reduce stack usage, maybe even a questionable performance
  * optimization.
  *
- * rusEfi main processing happens on IRQ so PORT_INT_REQUIRED_STACK has to be pretty large. Problem
```

Comment:
Please replace this comment not just remove this comment

```Interrupts are NOT serviced on the stack of the thread that was running when the interrupt occurred. The only thing that happens on that thread's stack is that its registers are pushed (by hardware) when an interrupt occurs, just before swapping the stack pointer out for the main stack (currently 0x400=1024 bytes), where the ISR actually runs.``` is good content to maybe place here?